python/test/unit/test_link.py

```diff
@@ -0,0 +1,49 @@
+import sys
+
+import pytest
+import torch
+import triton
+import triton.language as tl
+from triton.language.extra import libdevice
+
+from triton._C.libtriton import llvm
+
+
+@triton.jit(noinline=True)
+def add_one(x_ptr, SQRT: tl.constexpr) -> None:
+    x = tl.load(x_ptr)
+    if SQRT:
+        x = libdevice.sqrt(x)
+    tl.store(x_ptr, x + 1.0)
+
+
+@triton.jit
+def add_one_indirect(x_ptr, SQRT: tl.constexpr) -> None:
+    add_one(x_ptr, SQRT)
+
+
+@pytest.mark.parametrize("use_libdevice", (False, True))
+@pytest.mark.parametrize("kernel", (add_one, add_one_indirect))
+def test_link_extern_libs(use_libdevice, kernel):
+    if not torch.cuda.is_available():
```

Comment:
I think this actually runs it on AMD side too given in PyTorch HIP "pretends" to be CUDA. I think you can just drop this check entirely.

Reply:
Done.